php/lang/security/injection/printed-request.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+// example key-value: name=%3Cscript%3Econfirm%28%29%3C%2Fscript%3E
+function dangerousEchoUsage() {
+    $name = $_REQUEST['name'];
+    // ruleid: printed-request
+    print("Hello : $name");
+    // ruleid: printed-request
+    print("Hello : " . $name);
+}
+
+function safeEchoUsage() {
+    $name = $_REQUEST['name'];
+    // ok: printed-request
+    print("Hello : " . htmlentities($name));
+}
+
+function doSmth() {
+    $name = $_REQUEST['name'];
+    // ruleid: printed-request
+    print("Hello :".$name);
+}
+
+function doSmth2() {
+    // ruleid: printed-request
+    print("Hello ".$_POST['name']." !");
+}
+
+function doSmth3() {
+    $name = $_GET['name'];
+    if (str_contains($name, 'foobar')) {
+        // ruleid: printed-request
+        print("Hello :".$name);
+    }
+}
+
+function doSmth4() {
+    // ruleid: printed-request
+    print("Hello ".htmlentities($_POST['name'])." !".$_POST['lastname']);
+}
+
+function doSmth5() {
+     // ruleid: printed-request
+    print("Hello ".trim($_POST['name']));
+}
+
+function doSmth6() {
+    $VAR = $_GET['someval']
```

Comment:
typo, missing semicolon
```suggestion
    $VAR = $_GET['someval'];
```